README.md

```diff
@@ -1,7 +1,7 @@
 # Watson Developer Cloud Go SDK v2.0.2
 [![Build Status](https://travis-ci.org/watson-developer-cloud/go-sdk.svg?branch=master)](https://travis-ci.org/watson-developer-cloud/go-sdk)
 [![Release](https://img.shields.io/github/v/release/watson-developer-cloud/go-sdk)](https://github.com/watson-developer-cloud/go-sdk/releases/latest)
-![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/watson-developer-cloud/go-sdk)
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/watson-developer-cloud/go-sdk/v2)
```

Comment:
```suggestion
![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/watson-developer-cloud/go-sdk?filename=v2/go.mod)
```
If you use the Gitub UI to look at your feature branch, you can see that the badge doesn't get rendered correctly.  I think if you change the url per my suggestion above it will work.